packages/zend-rest/composer.json

```diff
@@ -10,7 +10,9 @@
         "ext-dom": "*",
         "ext-reflection": "*",
         "ext-simplexml": "*",
+        "zf1s/zend-controller": "^1.15.3",
```

Comment:
could we keep `zend-controller` in `suggest`, please? It's required only by `Zend_Rest_Controller` and `Zend_Rest_Route`. So it's optional when only client/server classes are used.

Reply:
This was indeed a close call. I myself only use Zend_Rest_Controller and Zend_Rest_Route (and not the client/server classes). Both uses are indeed completely separate. `suggest` sounds better.

Reply:
@falkenhawk Can all other deps also be moved to suggest? With the exception of `Zend_Exception`. That makes sense when using only the MVC classes (Zend_Rest_Controller and Zend_Rest_Route).